pkg/api/pod/util.go

```diff
@@ -249,6 +249,12 @@ func DropDisabledAlphaFields(podSpec *api.PodSpec) {
 	for i := range podSpec.InitContainers {
 		DropDisabledVolumeMountsAlphaFields(podSpec.InitContainers[i].VolumeMounts)
 	}
+	for i := range podSpec.Containers {
+		DropDisabledVolumeDevicesAlphaFields(podSpec.Containers[i].VolumeDevices)
```

Comment:
can you add it into the existing loops instead?

Reply:
oh duh :-)  of course